### ISSUE TITLE
Feature request: option to disable hiding of media marked as Sensitive

### ISSUE THREAD

**Issue by CDR-Aurora** [ReticentJohn/Amaroq#100]
Despite having "Media Display" set as "Always show media marked as sensitive on the Mastodon preferences website, content will still appear as hidden on the Amaroq app. There is no clear setting for this either. If the app is designed to inherit the Mastodon preferences, it is failing to do so.

Amaroq app version 1.1.12 (636)

**Comment by TheBrain0110** [comment]
For even more fun, Mastodon has *completely separate* functions for "Mark media as sensitive" and "Hide post behind content warning", with related settings to show them. It's a bit of a mess really.

I'm not sure that Amaroq respects either of those settings, it just always hides everything.

**Comment by heavyrubberslave** [comment]
I agree. Would be a great thing to have an option to disable sensitive content warning completely. So having a flag to set in the app settings whether it should warn about sensitive content (blur images) or not:

https://github.com/ReticentJohn/Amaroq/blob/master/DireFloof/DWTimelineTableViewCell.m#L549

**Comment by rightmeow666** [comment]
Alright, see if this is what you guys are looking for.
https://github.com/ReticentJohn/Amaroq/pull/116
@heavyrubberslave @TheBrain0110 @CDR-Aurora

**Comment by rightmeow666** [comment]
Hey @ReticentJohn, would this still a feature the app is looking to add. For what I am see so far, only me and a couple of guys above found it useful. (Escpecially at home or anywhere that's not a workplace.)

**Comment by ReticentJohn** [comment]
Hey @rightmeow666 - thanks for the PR, and apologies for the delay in review! I’ll see about merging this in over the weekend.
